src/components/Sum.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({
+  a = 2,
+  b = 1,
+}) => (
```

Comment:
`a` and `b` don't need to have random default values.
You already declare them as `required` in `propTypes`.